File: Scala/ProblemSolving/Beautiful Days at the Movies.scala
import java.io._
import java.math._
import java.security._
import java.text._
import java.util._
import java.util.concurrent._
import java.util.function._
import java.util.regex._
import java.util.stream._
import scala.collection.immutable._
import scala.collection.mutable._
import scala.collection.concurrent._
import scala.concurrent._
import scala.io._
import scala.math._
import scala.sys._
import scala.util.matching._
import scala.reflect._

object Result {


  def beautifulDays(i: Int, j: Int, k: Int): Int = {
    var count = 0
    for(n <- i to j){
      val reversenum = n.toString.reverse.toInt
      val diff = (reversenum - n).abs
      if (diff % k == 0){
        count = count + 1
      }
    }
    count
  }

}

object Solution {
    def main(args: Array[String]) {
        val printWriter = new PrintWriter(sys.env("OUTPUT_PATH"))

        val firstMultipleInput = StdIn.readLine.replaceAll("\\s+$", "").split(" ")

        val i = firstMultipleInput(0).toInt

        val j = firstMultipleInput(1).toInt

        val k = firstMultipleInput(2).toInt

        val result = Result.beautifulDays(i, j, k)

        printWriter.println(result)

        printWriter.close()
    }
}

File: Python/ProblemSolving/Beautiful Days at the Movies.py
#!/bin/python3

import math
import os
import random
import re
import sys


def beautifulDays(i, j, k):
    count = 0
    for i in range(i, j + 1):
        reverse = str(i)[::-1]
        diff = abs(i - int(reverse))
        if diff % k == 0:
            count = count + 1
        
    return count

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    first_multiple_input = input().rstrip().split()

    i = int(first_multiple_input[0])

    j = int(first_multiple_input[1])

    k = int(first_multiple_input[2])

    result = beautifulDays(i, j, k)

    fptr.write(str(result) + '\n')

    fptr.close()
